docs/pages/server-access/guides/ec2-discovery.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: Configure Teleport to Automatically Discover EC2 instances
+description: How to configure Teleport to install itself EC2 instances.
+---
+
+Teleport's SSH Service can be configured to automatically discover, and install
+Teleport on newly provisioned EC2 nodes.
```

Comment:
Can we add one or two more sentences here the describes how it works? Something like:

A Teleport Process can be configured to run a discovery service, which connects to EC2, discovers instances, and runs an install script on these instances using SSM"